src/llmq/instantsend.cpp

```diff
@@ -615,13 +616,11 @@ bool CInstantSendManager::CheckCanLock(const COutPoint& outpoint, bool printDebu
         return false;
     }
 
-    const CBlockIndex* pindexMined;
-    int nTxAge;
-    {
+    auto [pindexMined, nTxAge] = [&]() {
         LOCK(cs_main);
-        pindexMined = m_chainstate.m_blockman.LookupBlockIndex(hashBlock);
-        nTxAge = m_chainstate.m_chain.Height() - pindexMined->nHeight + 1;
-    }
+        auto pindex = m_chainstate.m_blockman.LookupBlockIndex(hashBlock);
+        return std::make_pair(pindex, m_chainstate.m_chain.Height() - pindex->nHeight + 1);
+    }();
```

Comment:
looks unrelated

Reply:
but is it bad? Yeah... It is pretty unrelated it seems.